dev-python/svg-path/svg-path-3.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+MY_PN="svg.path"
+
+DESCRIPTION="SVG path objects and parser"
+HOMEPAGE="https://github.com/regebro/svg.path"
+SRC_URI="https://github.com/regebro/svg.path/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+MODNAME="svg"
+
+python_test() {
+	${EPYTHON} setup.py test || die
+}
+
+src_install() {
```

Comment:
Err… why can't you let setup.py install it? Is it because of namespaces?

Reply:
Yes, `setup.py` tries to do namespace magic.